src/functions.js

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-use-before-define */
+/* eslint-disable brace-style */
```

Comment:
- [ ] Kindly remove the lines that disable linters in your JS files, I see that you're doing it a lot and I advise you against it since it's part of every project requirement to respect linters rules and fix them because they force us to use best practices and to write clean codes 👍